packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -109,6 +170,10 @@ mixin SearchControllerMixin<T extends DataSearchStateMixin> {
     onMatchChanged(activeMatchIndex);
   }
 
+  /// Delay to reduce the amount of search queries
+  /// Duration.zero (default) disables debounce
+  Duration get debounceDelay => Duration.zero;
```

Comment:
Instead of returning Duration.zero, why not just return null? Since we are never actually waiting for a Duration of zero, null seems like it would better indicate the state of "no value" or disabled